.github/workflows/coverage.yml

```diff
@@ -0,0 +1,54 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+jobs:
+
+  coverage:
+    name: Coverage (+nightly)
+    timeout-minutes: 30
+    runs-on: ubuntu-20.04
```

Comment:
Should we leave this on `-latest`?

Reply:
We can try, i am not sure if we are using ubuntu 20 here for some reason.

I saw some warnings in actions about this so all latest will be in ubuntu-20 soon:

> Ubuntu-latest workflows will use Ubuntu-20.04 soon. For more details, see https://github.com/actions/virtual-environments/issues/1816

Reply:
Trying with https://github.com/ZcashFoundation/zebra/pull/1758/commits/5edc34d91c7ee2204c4a75c331934e0669e227ab

Reply:
Sure but we don't want to pin on -20.04, we want to keep up with -latest on each platform

Reply:
Definitely agree